.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,33 @@
+name: Static analysis
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  push:
+    branches: [9.x]
+jobs:
+  static-analysis:
+    name: Static analysis
```

Comment:
minor - but the title is wrong. running unit tests is not considered static analysis. Let's call it "PHPUnit" or "Unit tests"